docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '2.0'
+services:
+  mongodb:
+    image: mongo:3.2.17
+    environment:
+      - MONGO_DATA_DIR=/data/db
+      - MONGO_LOG_DIR=/dev/null
+    volumes:
+      - ./data/db:/data/db
+    ports:
+      - 27017:27017
+  umdio:
```

Comment:
I think I should probably set the MONGO_RUBY_DRIVER_HOST environment variable in here, since it needs to be mongodb instead of localhost

Reply:
Done